akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorSpec.scala

```diff
@@ -247,26 +264,28 @@ object EventSourcedBehaviorSpec {
                                                         .thenStop
 
                                                     case Fail =>
-                                                      throw new TestException("boom!")
+                                                      throw TestException("boom!")
 
                                                     case StopIt =>
                                                       Effect.none.thenStop()
 
                                                   },
-                                                eventHandler = (state, evt) ⇒
+                                                eventHandler = (state, evt) =>
                                                   evt match {
-                                                    case Incremented(delta) ⇒
+                                                    case Incremented(delta) =>
                                                       probe ! ((state, evt))
                                                       State(state.value + delta, state.history :+ state.value)
-                                                  }).receiveSignal {
-      case RecoveryCompleted(_) ⇒ ()
-      case SnapshotCompleted(_) ⇒
-        snapshotProbe ! Success(Done)
-      case SnapshotFailed(_, failure) ⇒
-        snapshotProbe ! Failure(failure)
-    }
+                                                  })
+      .receiveSignal {
+        case RecoveryCompleted(_) ⇒ ()
+        case SnapshotCompleted(_) ⇒
+          snapshotProbe ! Success(Done)
+        case SnapshotFailed(_, failure) ⇒
+          snapshotProbe ! Failure(failure)
+       case e: RetentionSignal =>
+         retentionProbe ! Try(e)
```

Comment:
No need to wrap it in a `Try` if it always will be a success

Reply:
Good catch, totally missed that one.